hack/verify-bazel.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
```

Comment:
Is this just a handy script to have or are there other plans for this script e.g. CI? We may want to consider adding it to the development doc.

Reply:
Yes, the idea is to be able to run it in CI.

Agreed that it should be added to the docs. I think I'd like to do a doc fixup after all these PRs are in, rather than updating the docs multiple times in the middle of reviews, if that's OK with you.